vite.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vite'
```

Comment:
**praise**: Good job importing the necessary modules for Vite configuration. This is a clean and efficient setup.